spec/support/pdf_matchers.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rspec/expectations"
+require "pdf-reader"
+
+RSpec::Matchers.define :match_pdf_content do |fixtures_path_to_expected_pdf|
+  match do |actual|
+    expected = PDF::Reader.new(fixtures_path_to_expected_pdf)
+    @expected = expected.pages.map(&:text).join
+    @expected = @expected.delete("\n").delete(" ").delete("\t")
+
+    temp_pdf = Tempfile.new("pdf")
+    temp_pdf.binmode
```

Comment:
I had to go away and google what `temp_pdf.binmode` does, though having done so it now makes perfect sense. As there will be future new devs who might come across this, and it looks a bit random (`binmode.include?(:swing, :pedal, :touch)` 😁 ) could we get a brief comment on what it is and why it's needed?

Reply:
Of course! Is mostly to deal with the way the PDF is generated and store in the different OS. I'll add a comment :D